website/source/docs/agent/configuration/telemetry.html.md

```diff
@@ -114,6 +114,15 @@ telemetry {
 }
 ```
 
+### `prometheus`
+
+These `telemetry` parameters apply to
+[Prometheus](https://prometheus.io).
+
+- `prometheus_metrics` `(bool: false)` - Specifies whether the agent should
+  initialize a PrometheusSink, where metrics will be reachable via the metrics
```

Comment:
Specifies whether the agent should make Prometheus formatted metrics available at `/v1/metrics...`